editor/block-settings-menu/index.js

```diff
@@ -39,6 +39,12 @@ function BlockSettingsMenu( { onDelete, onSelect, isSidebarOpened, toggleSidebar
 				icon="trash"
 				label={ __( 'Delete the block' ) }
 			/>
+			<IconButton
+				className="editor-block-settings-menu__control"
+				onClick={ onToggleMode }
+				icon="html"
+				label={ __( 'Switch between the visual/textual mode' ) }
```

Comment:
TIL "textual" is a word - still think "HTML" or "text" might be clearer.

Reply:
>TIL "textual" is a word - still think "HTML" or "text" might be clearer.

Still think this ought to be addressed

Reply:
sorry forgot about this, updating

Reply:
going for "text" for consistency with the global mode switcher